File: 프로그래밍/c/code/05.전체.c
#include <stdio.h>

int main(void) {
    int n;
    int sum = 0;

    printf("몇 번째 수까지 더할까요? ");
    scanf("%d", &n);

    for (int i = 1; i <= n; i = i + 1) {
        sum = sum + i;
    }

    printf("1부터 %d까지의 합은 %d, 평균은 %.2f입니다.\n", n, sum, (double) sum / n);
    return 0;
}

File: 프로그래밍/py/code/05.전체.py
scores = list(map(int, input("점수를 빈칸으로 나누어 입력하세요: ").split()))

print("입력한 점수:", scores)
print("개수:", len(scores))
print("합계:", sum(scores))
print("평균:", sum(scores) / len(scores))
print("가장 높은 점수:", max(scores))
print("낮은 순으로:", sorted(scores))
